src/runtime/account.ts

```diff
@@ -209,5 +300,105 @@ export class ContractState {
     }
     return value.toJSON();
   }
+}
+
+export class Transaction {
+  private actions: Action[] = [];
+  protected receiverId: string;
+  constructor(protected sender: Account, receiver: Account | string) {
+    this.receiverId =
+      typeof receiver === "string" ? receiver : receiver.accountId;
+  }
 
+  addKey(publicKey: string | PublicKey, accessKey: AccessKey = fullAccessKey()): Transaction {
+    this.actions.push(addKey(PublicKey.from(publicKey), accessKey));
+    return this;
+  }
+
+  createAccount(): Transaction {
+    this.actions.push(createAccount());
+    return this;
+  }
+
+  deleteAccount(beneficiaryId: string): Transaction {
+    this.actions.push(deleteAccount(beneficiaryId));
+    return this;
+  }
+
+  deleteKey(publicKey: string | PublicKey): Transaction {
+    this.actions.push(deleteKey(PublicKey.from(publicKey)));
+    return this;
+  }
+
+  async deployContractFile(code: string | Buffer | Uint8Array): Promise<Transaction> {
+    return this.deployContract(typeof code === 'string' ? await fs.readFile(code) : code);
+  }
+
+  deployContract(code: Uint8Array): Transaction {
+    this.actions.push(deployContract(code));
+    return this;
+  }
+
+  functionCall(
+    methodName: string,
+    args: object | Uint8Array,
+    {
+      gas = DEFAULT_FUNCTION_CALL_GAS,
+      attachedDeposit = NO_DEPOSIT,
+    }: { gas: BN | string; attachedDeposit: BN | string }
+  ): Transaction {
+    this.actions.push(
+      functionCall(methodName, args, new BN(gas), new BN(attachedDeposit))
+    );
+    return this;
+  }
+
+  stake(amount: BN | string, publicKey: PublicKey | string): Transaction {
+    this.actions.push(stake(new BN(amount), PublicKey.from(publicKey)));
+    return this;
+  }
+
+  transfer(amount: string | BN): Transaction {
+    this.actions.push(transfer(new BN(amount)));
+    return this;
+  }
+
+  // TODO: expose signAndSend in naj
+  /**
+   * 
+   * @param keyPair Temporary key to sign transaction
+   * @returns 
+   */
+  async signAndSend(keyPair?: KeyPair): Promise<FinalExecutionOutcome> {
+    let oldKey: KeyPair;
+    if (keyPair) {
+      oldKey = await this.sender.getKey(this.sender.accountId);
+      await this.sender.setKey(this.sender.accountId, keyPair);
+    }
+    // @ts-expect-error
```

Comment:
linter is going to want you to say what error you expect at the end of this comment